.circleci/scripts/python_doc_push_script.sh

```diff
@@ -50,21 +50,17 @@ export PATH=/opt/conda/bin:$PATH
 
 rm -rf pytorch || true
 
-# Install TensorBoard in python 3 so torch.utils.tensorboard classes render
-pip install -q https://s3.amazonaws.com/ossci-linux/wheels/tensorboard-1.14.0a0-py3-none-any.whl
-
 # Get all the documentation sources, put them in one place
 pushd "$pt_checkout"
-checkout_install_torchvision
+# Do not do this, since torchvision is a separate project.
+# checkout_install_torchvision
 pushd docs
-rm -rf source/torchvision
-cp -a ../vision/docs/source source/torchvision
+# rm -rf source/torchvision
+# cp -a ../vision/docs/source source/torchvision
```

Comment:
torchvision is building its own documentation at https://pytorch.org/vision. This was fixed in gh-41334

Reply:
Do you need to keep those lines as comments rather than just deleting them?

Reply:
removing